css/src/components/form/input.scss

```diff
@@ -0,0 +1,84 @@
+$input-color: $text !default;
+$input-background-color: $body-background !default;
+$input-border-color: $text !default;
+$input-placeholder-color: $text-subtle !default;
+
+$input-hover-color: $text !default;
+$input-hover-border-color: $primary !default;
+$input-focus-border-color: $primary !default;
+
+$input-disabled-color: $text-subtle !default;
+$input-disabled-background-color: $body-background-medium !default;
+$input-disabled-border-color: $table-border-dark !default;
+
+$input-error-border-color: $danger !default;
+$input-warning-border-color: $warning-active !default;
+$input-success-border-color: $score-high !default;
+
+$input-focus-box-shadow-size: 0 0 0 0.125rem !default;
```

Comment:
should the box shadow size be tied to the elemnts font size? `0.125em`?

Reply:
I don't really see a lot of benefits from it tbh. Also, the outline rules we have set up in `rem` too, so I would prefer to keep the same way for consistency.

Reply:
okay, cool, thanks for testing!